packages/playground/windows/playground/MainPage.cpp

```diff
@@ -34,7 +34,14 @@ void MainPage::OnLoadClick(
     Windows::Foundation::IInspectable const & /*sender*/,
     xaml::RoutedEventArgs const & /*args*/) {
   auto host = Host();
-  auto bundleFile = unbox_value<hstring>(x_entryPointCombo().SelectedItem().as<ComboBoxItem>().Content());
+
```

Comment:
the PR title says you're fixing e2e test but this is changing playground?

Reply:
This is a minor fix to allow playground to load bundles outside of the hard coded list.
-- I can open this as a separate PR if you want